programs/survival/README.md

```diff
@@ -139,10 +139,9 @@ Various scripts that modify various game elements, often replicating popular mod
 ### replace_inventory.sc:
 #### By gnembonmc
 	There is a video on his channel about this.
-	If you run out of an item for any reason, it will be refilled from the inventory, then shulker boxes.
-	Useful for large scale building, or afk sand placing around a monument.
+	Allows a players inventory slots to be filled when they run out with same blocks from a shulker box in the players inventory which makes it useful for
```

Comment:
```suggestion
	Allows a players inventory slots to be filled when they run out with same blocks from a shulker box in the players inventory, which makes it useful for
```

Reply:
I also think it needs some pauses, and maybe the old description was better, since it also mentioned refilling from the inventory.

Reply:
also it's not just blocks which cause refilling, it's any old item (eg. rockets)

Reply:
I have made the suggested changes. Thanks